src/Sylius/Bundle/ApiBundle/Serializer/ImageNormalizer.php

```diff
@@ -40,7 +40,7 @@ public function __construct(
 
     public function normalize($object, $format = null, array $context = [])
     {
-        Assert::isInstanceOf($object, ImageInterface::class);
+        Assert::isInstanceOf($object, Image::class);
```

Comment:
I'm not sure if this change will resolve the initial problem, because still if `path` is not serialised then `$data['path']` will be empty, we should probably omit the `resolvePath` method when the path is not serialised and leave the interface as it was 🤔